.gitignore

```diff
@@ -45,4 +45,5 @@ Thumbs.db
 
 # Next.js
 .next
-out
\ No newline at end of file
+out
+vitest.config.*.timestamp*
```

Comment:
What's this for? :raised_eyebrow:

Reply:
Unsure. I noticed such file was temporarily when I built and ran tests. This line was generated by Nx plugin.